exp/rendering-architecture.md

```diff
@@ -15,6 +17,16 @@ For AMD and Intel GPUs, Anbox Cloud uses the [Mesa driver](https://www.mesa3d.or
 
 See [Component versions](https://anbox-cloud.io/docs/component-versions) to refer to the actual version supported for any particular Anbox Cloud release.
 
+# Supported platforms
+
+For rendering, you can use the `swrast` or the `null` platforms depending on your requirements.
+
+`swrast` is a software rasterization platform, which is a rendering implementation of the Mesa driver with support for LLVMpipe. It can be utilised for use cases that require a visual output without a GPU. When you launch an Anbox container with the `swrast` platform using the `amc launch -p swrast <app_id>` command, the rendering pipeline uses this platform irrespective of any available GPUs. The rendering pipe for the `swrast` or `null` platform is not different than the one for the `webrtc` platform with NVIDIA GPU support except for the fact that it's irrespective of any available GPUs. To know more about this implementation, see [LLVMpipe](https://docs.mesa3d.org/drivers/llvmpipe.html).
+
+`null` is an OpenGL headless platform that makes use of the rendering backend of the [Almost Native Graphics Layer Engine (ANGLE)](https://chromium.googlesource.com/angle/angle) and can be used when you do not need a graphic output, such as, automation testing.
```

Comment:
I feel this doesn't emphazize enough that the `null` platform does not produce any pixel output or even perform software rendering. We should also highlight that it has very low overhead on the CPU is with that perfect for all use cases where no graphical output is necessary.